### ISSUE TITLE
Lots of "Maximum number of unsubmitted assignments" reports since Friday

### ISSUE THREAD

**Issue by pscohn** [rainforestapp/tester-chrome-extension#17]
Official issue for this problem many testers have reported in support. Many claim they can't get a single job since Friday, or could get only 2 or 3.

For many testers, this will be solved by using only one account for Rainforest tasks.

**Comment by begsoon** [comment]
Hello.
Since  Friday I only  do 5-7 hits.
Otherwise, I have worked a lot more because I really love this job on CrowdFlower.Yea,this problem start at Friday after I get bonuses 10% at the end of working day.
I need 2-3 hours to get only 1 Or 2..and then get this message ''You have reached the maximum number of unsubmitted assignments. So we are fair to other workers, we stop you doing too much for us at once. You can complete the other assignment(s) and refresh this page, or simply return this hit.'' 
Also I get that message even I didn't open a hit..
I don't give up on the job or something alse to get that message..this really strange what happend these day.


Best regards,
Begsoon

**Comment by nihlathar** [comment]
You know it's most likely the same as before, with this being a problem affecting most accounts, so you need to find a solution without the need for everyone to message you their id.

The day after you last fixed it, I was able to do about 15 jobs more than any time in the past. Day after that it started having trouble, the tab opens at least 1-2 times per hour but only few and far between are actually working, that is, terminal loads and task can be completed and submitted.

I do use the extension, I only have (and ever used) one/same account. Worker ID is 28155953

**Comment by Rainforestuser790** [comment]
Hello,

i was able to do just 7 hits in total from the day  the change was implemented and before that i was able to do close to 20 + hits daily and i am using both extension and CF id as well, but lately i always get the message  "unsubmitted assignments" for both of them when ever there is work available and i could say this as people mention the same in the blogs or fourms that they are able to work on a particular type of sites and the oops message changes to the unsbumitted assigneents. Today i could hardly get any hit just had couple of oops opened from extension and they to at some point give the same message .

i have no multiple accounts and i just joined last month , following is my Id # 34408790

Thanks

**Comment by Unsubmitted** [comment]
My worker ID  is # 31970468 and now i am getting very low hits  with the last change which you have implemented. I am using extension and 99% of the time it says the problematic message unsumbitted  The percentage falled down to almost 90% what i was able to do before. I am not having any multiple accounts nor worker ids. My channel is Clixsense.

Request you to please fix the issue before the testers lose interest in RF.

Thanks

**Comment by ukd1** [comment]
Hey all, part of the reason is a single human using multiple accounts - we're cracking down on this behavior (hoading / pre-loading work / holding work), which is part of what is happening to most of you. If you list your account ID here, we'll investigate each to find out what's going on.

Russ

**Comment by mueller2511** [comment]
Hi Russ,

I am using Rainforest tasks with my CF account with the following worker id #34225825 and i am facing the same issue like others and if you are mentioning that using multiple account for same person is creating  problem then i am sharing wifi with my husband who also does the RF tasks from his seperate CF account which is having different worked id , so you mean to say that this is also not allowed as we are both different person and not a single human being, in any case my husband has stopped the extension and powered of his pc and only i am doing and i still get the message of "unsubmitted message". 

Can you please look into the issue why i am still getting the same message when my husband has powered of his pc and not using any other means to do the task and vice versa he is also facing the same issue when i shutdown my pc and he is still not able to do the same way and he to keeps getting the message of unsubmitted message after doing 1 task or more.

Thanks

**Comment by Genesis2015** [comment]
In thepast I've used multiple accounts like 90% of the contributors from CF (considering that nowhere been written, it is prohibited) to performing tests, but since Friday I use only the extension. Now i am getting veееry low hits since Friday. I do not often meet the problem with "Maximum number of unsubmitted assignments" but this is my smallest problem. I completing test since more than an year and so far I have completed about 10,000 tests and I am one of the best testers here, but I am very discouraged to continue to work with you because it is not worth wasting my time just for few cents per day. You do not motivate people who contribute most to the system, just wondering how to allow the most people to join it - quality versus quantity! I do not think it is good for you and us! 
p.s.: Sorry for my English, but I am sure you will understand me. :)

**Comment by ando100** [comment]
@Genesis2015 -out of curiosity- if you feel sry for your English- how do you do the HITs? And yes, I do feel sry for your English as well.  Please let me know- where I or anyone can check- you are one the best testers here, thanks.

**Comment by m0dal** [comment]
My worker ID is # 33863320 and from Friday i did 3 hits. I recieve standard "Ooops" time to time, but more "Maximum number of unsubmitted assignments" (use extension and ID's from CrowdFlower). Don't work with multiply accounts.

**Comment by ando100** [comment]
@Genesis2015-- You deleted > you were using multiple accounts- from your message- any particular reason for that?

**Comment by mueller2511** [comment]
Right now my oops changed to the following message for your information and my husband laptop is off and he is not doing any RF task this moment.

You have reached the maximum number of unsubmitted assignments. So we are fair to other workers, we stop you doing too much for us at once. You can complete the other assignment(s) and refresh this page, or simply return this hit.

Thanks

**Comment by billu6628** [comment]
hello rainforest,
                          i have been doing your web application tasks from long back.i have never faced issues in such long time.all of sudden i started to get"Maximum number of unsubmitted assignments".i even disabled the extension but still it leads to the same.

my crowdflower id -27010801.please can you check the errors facing by me and clear me of it. i have never done anything to break your company policies.so looking forward to do some test for you.

thank you.

**Comment by disappointed** [comment]
#ando100 Instead sorry for other people it is better to learn English. There is a beautiful  word sorry, but the word SRY is only in your mind.

**Comment by rbasak2011** [comment]
Since friday i could do only 3-4 jobs.I am using chrome extension and i have only one account.My contributor id is: 32050036. 
My screen name is R.BASAK..When i got a hit from the extension its show this msg "Maximum number of unsubmitted assignments"..After refresh that 5-10 minutes its become an oops then after refreshing that oops for 2-3 minutes oops becomes this again "Maximum number of unsubmitted assignments"..Its so frustrating..

Thanks

**Comment by nerdatwork** [comment]
already emailed about the issue.

**Comment by rbasak2011** [comment]
yes i got the email but i thought its better to give my contributor id to
that official forum so i did that..

Regards,
Rabin Basak

**Comment by Savan47** [comment]
Same problem, never used other accounts or something like that. Active member for over 7 - 8 months. Please check me and solve this problem, thanks

Contributor ID: 32455098

Savan

**Comment by begsoon** [comment]
Still have same problem . 1 . Account . 1  Ops  refresh >  Unsubmitted assignments
:S

**Comment by Ranky2211** [comment]
Respected Russell Smith,

Hello

I have only one account and i am using extension.Tab opened by extension oops or unsubmitted assingnment or (HIT different no) cannot have new job assigned to it.After refreshing oops sometime got either HIT or You have reached the maximum number of unsubmitted assignments and i noticed that if i got that message and opened work from cf id it also gave the same unsubmitted assignments message.
I have also query about how much time these assisgnment id generated by extension or CF  valid for refreshing for getting HIT. 

Thanks

With Regard,

Worker id - 33432784

Ranky

**Comment by KabakF** [comment]
check me too please, same error

worker ID: 35273503

**Comment by Nuriii** [comment]
Facing the same problem my worker id is # 29451371.Thanks.

**Comment by tetanus1234** [comment]
since friday have same isue as others, my contributor ID 31807945

**Comment by ElPeter** [comment]
I'm using only 1 account, since today I didn't used the extension, I worked through the CrowdFlower, today there was no work so I installed the extension, I got 1 hit, completed it and after that I received this error...

I don't know why...

34460763 is my ID, if you can check it and fix it. Thank you.

**Comment by begsoon** [comment]
Yes,yes nobody now didn't use more accounts to do this job.Do not be liars!We all use that way to do tasks of Rainforest,and why we do that? Because we can do that and im sure Rainforest know for that.Now,when more people start to do tasks its to much for Rainforest and they can't support that.So we have now this situation,no more work on 2 3 accounts,just 1.Oke,no problem lets do it but first you need to reslove problem with this message ''Unsubmited Assigments. Only one Ooops  on refresh,One account and still got that message.Then I put only Extension,i cant get hit.Only Oooops or  Unsubmited Assigments.

I hope you will succeed in resolving this issue and we will continue to work on one account.
Maybe I will have problems  because I'm honest,but ok :)

Sorry guys for my english.

Gl in resolving this  issues.
Have a nice day.
Begsoon

**Comment by nerdatwork** [comment]
@begsoon I understand you have problem speaking/writing fluent English but calling Rainforest staff liars is not going to expedite(speed up) your issue. From what you have managed to type is that you used to use 2 or 3 accounts but now you use 1 so now some of us know how we could not get tasks. You aptly suit your nickname. Please refrain from name calling if you can.

**Comment by begsoon** [comment]
You don't understand me :D I don't call Rainforest staff a liars.I call these testers who posted that they don't do this job with 2 3 4 accounts :)

**Comment by nerdatwork** [comment]
@begsoon I stand corrected but in my humble opinion its not the place to call anyone liars. As @ukd1 said post your worker id besides that making other posts isnt going to help. It is only going to take more time for Rainforest staff to read through all these unnecessary comments. Did that make sense to you ?

**Comment by qpPykT** [comment]
Relax guys, why are you so angry?

**Comment by suja911** [comment]
hey regarding with all this what is happening,is it 17 hits a good number?

**Comment by chmi16** [comment]
35266086 CF elite
I have two other accounts on ClixSence 32880993 and Neobux 35169212 so you can block this two

**Comment by remo24** [comment]
Facing the same problem. I'm using only one account, my worker id 31841666.
When i got a hit from the extension it is showing "Maximum number of unsubmitted assignments" msg.
Please check and solve the issue.

**Comment by Sombordc** [comment]
I' having the same problem since Friday. My ID is 33128664 through CrowdFlower Elite channel. Message "You have reached the maximum number of unsubmitted assignments" is always present and i have managed to do only 5 tasks since Friday. Please review this ID.

**Comment by m0dal** [comment]
My worker ID is # 33863320 and from Friday i did 3 hits. I receive standard "Ooops" time to time, but more "Maximum number of unsubmitted assignments" (use extension and ID's from CrowdFlower).
I don't work with multiple accounts.

**Comment by qpPykT** [comment]
@m0dal You use one or more ID's from CrowdFlower?

**Comment by rnfrst** [comment]
To Russ,
In case you are reading this, I just wanna ask if you are aware that an account in Crowdflower (CF) allows the use of different channels (like Neobux, Clixsense, Crowdflower Elite, etc.)...  A user can opt to use these channels while still maintaining the same account provided that a Job ID should only be accessed in one channel (this is already controlled by CF system). The problem is that when a user access the "Test a Web Application" ID, a NEW Rainforest (RF) account is created for EACH CHANNEL. Users in this case do not mean to create a new RF account but it happens, they cannot control this... I think that's what you meant by using multiple accounts... If you want to stop this, then it's better to have a system in place preventing creation of multiple accounts in different channels.

As with the RF extension, if a user wants to work on his preferred channel, it can be impossible to do that because the extension is recognizing only the latest claimed profile (please take note that  the RF Tester Portal doesn't give an error when the profile is claimed by the same email account thereby disabling access to the other profile).  And when a user accessed a CF Job ID from different channel, it will definitely be recognized by RF system as using multiple account. 

I hope this will enlighten the situation of some RF testers and you could make the necessary adjustment. Many accounts are already affected by this changes... By the way, the worker ID is the same as the Contributor ID in CF so it is not advisable that it is visible to everyone.

**Comment by blago1989** [comment]
Hello, in the past few days I can not do any test in Rainforest, I have the same problem like others - my worker ID is 32663317

**Comment by llichigoll** [comment]
Hi ,

I need help, I'm not able to do a job for more than 15 hours while the computer, I'm always getting error messages such as:

"You have reached the maximum number of unsubmitted assignments. So we are fair to other workers, we stop you doing too much for us at once. You can complete the other assignment(s) and refresh this page, or simply return this hit."

or

"Ooops: another worker accepted the job before you! It doesn't look like there is any more work for any of our hits; please try again shortly - you can refresh this page, or return this hit."

could you please help me, I love working for rainforest and want to work more, my profile is :

https://portal.rainforestqa.com/tester/3282f6ab-2835-4b04-bea4-14ed1b3a0b1e

WORKER ID Crowdflower :
34655816 Clixsense

Thank you for help! :)

**Comment by elip0** [comment]
Hello,

I have only one account, my ID is 31001914.  

Only 1 hit since Friday and I keep getting "You have reached the maximum number of unsubmitted assignments..." 

I use the extension. I tried removing it, but it was the same without it.

Thank you

**Comment by ando100** [comment]
@disappointed, Thank you for your correction, English language is beautiful, then there are Shakespeare, Milton, Lord Byron etc. But you`re totally offtopic. Back to topic: This guy edited his message several times and I did reply to his- I assume 1st message. He was boiling and insulting all over+ included how he cheated. My message might have sounded harsh to you, but time-when you saw it- he already edited his message at least 3 times. I saw it 1st hand and replied to that. Thanks.

**Comment by ando100** [comment]
went through all messages, @begsoon sounds similar to Genesis2015 :) 
my bottomline as of today- the extension works and works good. Really! I do not have multiple accounts etc.,
got and did lots of tasks. Hoping the extension get the sound.

**Comment by mueller2511** [comment]
Hi Rainforest Team,

Is the issue identified or still yet to know the exact cause of the problem, as i am still getting the message.

Hoping you to fix the issue faster and have a nice day.

Thanks

**Comment by mueller2511** [comment]
ando100 it is good to see it is working fine for you but this thread is for the issue which is causing most of the testers not able to work and not for asking whether the extension is working fine or not, so please don't spam the thread and make it hard for the RF team to read the comments and think that the problem is gone.

I hope you understand what i mean to say.

Thanks

**Comment by ando100** [comment]
@mueller2511 There is no issue atm. 
Do you mean that:  Right now my oops changed to the following message for your information and my husband laptop is off and he is not doing any RF task at this moment.
-
You have reached the maximum number of unsubmitted assignments. So we are fair to other workers, we stop you doing too much for us at once. You can complete the other assignment(s) and refresh this page, or simply return this hit.

is that your issue?
This has been covered. 

or:
for the issue which is causing most of the testers not able to work

using multple accounts? come on.
really.
Am I wrong?

**Comment by mueller2511** [comment]
i am not saying you are wrong but don't spam the posts with other users details seems like you are having some issue with your attitude and highlighting some other posts shows it and i am not in the mood to spam the topic more.

**Comment by ando100** [comment]
no offence- I-´ve been around and "you and your husband" figure around in forums, I do not mind.
I do not care.. I was trying to help- as this as you describe is not allowed- thats it. Did not Spam, got the info from your username. All the best !

**Comment by mueller2511** [comment]
@ando100 lets stops fighting and just tell me whether you were having the same issue before or never got the message , if you did have the issue was it re-occurring to you lately or gone.

**Comment by ando100** [comment]
I never got the message.. And yes- all working like a charm. Might be- one person-me- using one account, one laptop etc. Peace and no hard feelings, all ok and great :) RF now got the forum they were looking for, lol :)

**Comment by mueller2511** [comment]
Good to most of you are not having issue but speaking as of now i am getting the message 
"You have reached the maximum number of unsubmitted assignments. So we are fair to other workers, we stop you doing too much for us at once. You can complete the other assignment(s) and refresh this page, or simply return this hit."

following is the assignment id

https://tester.rainforestqa.com/tester/assignment/crowd_flower/any?assignmentId=cxecf990-ac48-4cd9-9449-b8077230a1a0&browser=any&hitId=4550964&sig=a7eda4d1e5ad7d251943eb39e7232e11d9399a5f44c03649f60098499c79498b&turkSubmitTo=%2Fthanks&workerId=34225825

Thanks

**Comment by tetanus1234** [comment]
same message all day, did anyone find any solution???

**Comment by KabakF** [comment]
I think that they should write some code like "This job only can be completed by one user for this IP address" 
So people like those guys who uses multiple accounts on same time would be sad for more fair working terms.

**Comment by Savan47** [comment]
I can't believe how KabakF and others who post here and don't have any problem, can be so selfish. If some of us told that we don't use any other accounts and we still get same message, why would you again start to insult us?
Now we all here lie and you are fair tester, wow great for you.

I personally didn't even gave my brother to jump in and work because of IP.

If it's working for you then everyone is unfair, if you've had the same problem then you would consider this a BUG?
This also can refer ando100.

It's simple if you don't have any problem with the bug mentoined in topic then don't post, please.

Thanks,
Savan

**Comment by KabakF** [comment]
@Savan47 hey hey slow down, I never said such a thing like you are liar. I only say It would be fair if everyone use one account to work on. I have same problem and I am not happy cuz you can't work or like I'll do your jobs too, just please don't make up.

I'm here to say my opinion and I think I have right to do that. I never have insulted someone here.

Thanks

**Comment by Savan47** [comment]
@KabakF 
Sorry but your "So people like those guys" had double meaning for me, now i see what you thought about.

It's not like i am furious or you ruined my day or i care if "my job goes to someone else" it's like there is to much spam about what should be good/fair and what is not.
I am not saying we shouldn't be fair to each others but that's not just topic about that + reading all this spam post will only make harder and longer for Rainforest to fix us up.

So basically if you have problem mentioned in topic leave ID here if you not then don't post.

And yes you have right for your opinion but by writing it you are just making things goes slower.

This is my opinion about you. (Consider this post spam but that's how it goes, spam always cause more spam) 
So whatever you write next i will not respond due to respect to other workers.

Have a nice day,
Savan

**Comment by KabakF** [comment]
@Savan47 I find you right and deleted my comments, I'll delete this one too, after you see it

**Comment by mueller2511** [comment]
@tetanus1234 , same issue for me still and i don't think they have found any solution.

**Comment by sizika66** [comment]
Hello! I have the same problem like others, since friday. "You have reached the maximum number of unsubmitted assignments..." The last HIT was: 2015-11-25 18:21:59 UTC Since than I can not do any test, I use extension and it's always "NO". I tried to do the task even without extension but it's the same problem. I've reinstalled extensions several times, again the same problem. I use Windows XP, SP3. Worker ID 
12316854 Crowdflower Neobux. Thank you! :)

**Comment by androwg** [comment]
@ukd1 Also i have this problem when any hit opens i got this message "You have reached the maximum number of unsubmitted assignments..."
i have no working in any hits and i dont have another account my worker id (32916100)

**Comment by ihorpolianetskyi** [comment]
again and again, I have only one hit which says: "You have reached the maximum number of unsubmitted assignments. So we are fair to other workers, we stop you doing too much for us at once. You can complete the other assignment (s ) and refresh this page, or simply return this hit. "can I do something wrong, employee ID Crowdflower:
33330543 ClixSense. Road Administration, please check it please.

**Comment by pscohn** [comment]
Hey Testers,

Just wanted to give you a quick update. We've found a potential fix for this problem and expect to have it out on Monday. I know this is a very frustrating issue for many of you, so thank you for your patience, and we hope to have it fixed as soon as possible!

Paul

**Comment by mueller2511** [comment]
Very happy to see that you have found the bug and going place a fix for it which makes me to forget all the stress which we had complete week.

Thanks RF Team.

**Comment by Ranky2211** [comment]
Hooray ! Hi  Paul Thanks.Please resolve also HIT xxxxxxx cannot have.new job assigned to it.I got hundred of it.If you need these hit number for resolving this issue.I can give you all hit number.Thanks for your concern.

With Regard,

Worker id - 33432784

Ranky

**Comment by Ranky2211** [comment]
Hi @pscohn and ukd1 Can you please tell me if i have doubt about rule posted on tester portal where can i clarify my doubt.I understand all rule except only one which was implemented by Russ.Thanks

**Comment by pscohn** [comment]
@Ranky2211 you can email our support channel at testers@rainforestqa.com

**Comment by Ranky2211** [comment]
@pscohn Thanks for reply.I already asked them but they reply all my query but not clarifying me about that rule.I think either they don't know properly about it like me or not understanding my question.Okay i asked them again.

**Comment by androwg** [comment]
@pscohn the only thing i hope that these problems doesnot affect our rate in Rf
Our Rate sufferend a lot from(terminal with no connection-we have reached our maxium-fail to take screenshot-security error occured Error #204-connection lost)

**Comment by pscohn** [comment]
@androwg The rating in your profile is actually not a very good representation of your accuracy, since it does include errors like you mentioned. We're going to change the way it's calculated to only include jobs that you've completed, so for now don't worry if it goes down due to errors :)

**Comment by NSPthakur** [comment]
Please post job IDs on task page. because job ID work better than Extension. And "you have reached the maximum" problem still persists.

**Comment by tetanus1234** [comment]
it is monday and same message still, can't work

**Comment by begsoon** [comment]
Its 06:33:06 PST in San Francisco :D

**Comment by ihorpolianetskyi** [comment]
Good day! This again I have nothing I have not changed: You have reached the maximum number of unsubmitted assignments. So we are fair to other workers, we stop you doing too much for us at once. You can complete the other assignment(s) and refresh this page, or simply return this hit. Job (hit):
https://tester.rainforestqa.com/tester/assignment/crowd_flower/any?assignmentId=cx1540b8-c4c3-4426-8a32-aa9c71259b89&browser=any&hitId=4651548&sig=ca3b0239d933f2145608711bf629fab80f690d8103a7d2b3bfea9448d7903d90&turkSubmitTo=%2Fthanks&workerId=33330543
Please administration dear, do something about it!

**Comment by ahmedofhell** [comment]
And..
Application Error
An error occurred in the application and your page could not be served. Please try again in a few moments.
If you are the application owner, check your logs for details.

Error message all the time!

**Comment by mueller2511** [comment]
Getting the message "You have reached the maximum number of unsubmitted assignments. So we are fair to other workers, we stop you doing too much for us at once. You can complete the other assignment(s) and refresh this page, or simply return this hit"

hoping it will get resolved by EOD today.

**Comment by pscohn** [comment]
The fix is still planned for release by EOD, though it **might** only be a temporary fix. We will have to see whether it solves the problem permanently.

**Comment by NSPthakur** [comment]
Thank you @pscohn . And when new ids will posted on task list?

**Comment by ahmedofhell** [comment]
Thank you @pscohn We appreciate your help.

**Comment by mueller2511** [comment]
@pscohn can you please confirm whether the fix has been implemented or yet to implement.

Thanks

**Comment by Savan47** [comment]
It works now, i am getting job one after another. 
How about you guys?

**Comment by chmi16** [comment]
Still same message. Only 2-3 hits per day

**Comment by ahmedofhell** [comment]
Yes It's works fine for me.

Thanks @pscohn for your help.

**Comment by sizika66** [comment]
Still same: : "You have reached the maximum number of unsubmitted assignments..." Just one HIT today. It's frustrating! :(

**Comment by ihorpolianetskyi** [comment]
The same as always, it's probably best for me. This is very sad.
Can the road administration and developer change or correct something ?!

**Comment by BV25** [comment]
@pscohn Hi Paul, 

I'd also like to bring to your attention that when this max assignment problem started, there's another issue which popped up - that of slow copy-pasting and/or typing inside the terminal. It is extremely slow to copy-paste any link or even type manually.

This issue had occurred in the last month too, but was resolved within 2 days that time. Now, it has been more than a week, I believe you might be unaware so I thought of informing you.

Could you please let the team know of this issue? Thanks in advance :)

**Comment by ahmedofhell** [comment]
"You have reached the maximum number of unsubmitted assignments" again!!

**Comment by pscohn** [comment]
The fix has been partially deployed as of last night - there was a problem implementing it, but it should be completed within a few hours.

**Comment by androwg** [comment]
it works fine for me now :)

**Comment by ihorpolianetskyi** [comment]
and for me. Thank You!

**Comment by sizika66** [comment]
It works now. Thank you! :)

**Comment by mueller2511** [comment]
working fine upto now without any problem. Thanks pscohn for implementing the fix.

**Comment by tetanus1234** [comment]
just wanted to say thank you, works PERFECT  :)

**Comment by pscohn** [comment]
Seems like it's working for most testers now, closing.

**Comment by tomychay** [comment]
I get the message: "You have reached the maximum number of unsubmitted assignments. Compared to other workers fair We Are, We stop you doing too much for us at once. You can complete the other assignment (s) and refresh this page, Simply return the this or hit. " many days. You can remove this notice or I could do to this message disappears from my tests, During peak hours of testing, I found utilities turned "ON" and then turn off, a period of time, the new test appear again. I have watched in a long time, Obviously I have to wait for other workers completed, I new can continue to do another test post. I sent a lot of support tickets but the problem is not solved
You take a look and help me solve the problem! Thanks!
tomychay / ID: 31605010

**Comment by nihlathar** [comment]
@tomychay that's intentional, you aren't supposed to work on more than 1 task at a time. So just make sure you have only 1 active HIT at a time, dont open other windows with 'taken' hits that you try to refresh, because that doesn't do anything, as soon as there's work available the extension will find it for you (they are all either available or unavailable at the same time, that's how it works). I'm sorry I couldn't put this more technically. Just don't try to outsmart the system :) I reinstall the extension periodically, when you see the flashing yes but no tab opening, or other issues, that's what solves it (I actually set up a bookmark and I just reinstall it after every hit (if nothing new comes in 1min), just to make sure
